libs/pages/application/ui/src/lib/components/container/container.tsx

```diff
@@ -50,6 +50,8 @@ export function Container(props: ContainerProps) {
           url: link.url || '',
           external: true,
         },
+        copy: link.url || '',
```

Comment:
Why not accepting null for copy? In this case if copy is null you just don't show nothing